eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -47,7 +47,11 @@ param (
     [string] $Environment = 'AzureCloud',
 
     [Parameter()]
-    [switch] $Force
+    [switch] $Force,
```

Comment:
I noticed that both New/Remove-TestResources have a Force switch however "-Force" is being used in a number of places which isn't being defined by the switch setting. Further, the switch is being set to when both scripts are being called so it begs the question of whether or not the parameter is actually necessary. Is there a case where we wouldn't want the -Force option?

Reply:
I'm inclined to agree that we should be using `-Force:$Force` instead of arbitrarily using `-Force` in the `Connect-AzAccount` command because

* This is a destructive action on current credentials (hence the previous proposal to delete existing credentials on Mac) so we want the user to make an informed decision here
* We're already using `-Force` in the CI templates so we won't see

Reply:
@weshaggard, @heaths would there be reasons against doing this?

Reply:
I'd be Ok with plumbing the force parameter through.

Reply:
I just said the same thing in review: `-Force:$Force`. Unless there's a good reason we should overwrite whatever the user has done (in this case, set a specific account/context), we shouldn't do it for them. That goes against recommended guidelines for powershell development.